app/server/appsmith-interfaces/src/main/java/com/appsmith/external/models/DecryptedSensitiveFields.java

```diff
@@ -0,0 +1,28 @@
+package com.appsmith.external.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+//This class should never be used except exporting the application as we need to export sensitive info in response
```

Comment:
Please confirm my understanding as well, written below 👇 😄

```suggestion
/**
 * This class hold sensitive information, and fields that have a `@JsonIgnore` on them, so that such information
 * can be serialized when an application is exported.
 */
```

Reply:
Yeah modifications are more appropriate compared to earlier comments!